packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/SVGPanZoom/SVGPanZoom.js

```diff
@@ -78,8 +77,8 @@ function SVGPanZoom({
     setViewBox((prevViewBox) => {
       const newViewBox = {
         ...prevViewBox,
-        x: prevViewBox.x - difference.x,
-        y: prevViewBox.y - difference.y
+        x: prevViewBox.x - difference.x / 1.5,
+        y: prevViewBox.y - difference.y / 1.5
```

Comment:
Changing the smoothing factor from 1.5 to 1.2 here gives better performance in Chrome on my Macbook, but I suspect this varies from device to device.